src/css/onboarding.css

```diff
@@ -0,0 +1,30 @@
+#container {
+    display: flex ;
+    justify-content: center ;
```

Comment:
Can you add the following here so that the image is not stretched vertically but centered? It slightly changes the position of the image, but I think it looks more like your reference image and more like the stuff isn't tappable anyway.

`align-items: center ;`